cli/script/management-sdk.ts

```diff
@@ -497,28 +501,57 @@ class AccountManager {
           const writeStream: fs.WriteStream = fs.createWriteStream(fileName);
 
           zipFile.outputStream
-            .pipe(writeStream)
-            .on("error", (error: Error): void => {
-              reject(error);
-            })
-            .on("close", (): void => {
-              filePath = path.join(process.cwd(), fileName);
-
-              resolve({ isTemporary: true, path: filePath });
-            });
+          .pipe(writeStream)
+          .on("error", (error: Error): void => {
+            reject(error);
+          })
+          .on("close", (): void => {
+            filePath = path.join(process.cwd(), fileName);
 
-          for (let i = 0; i < files.length; ++i) {
-            const file: string = files[i];
-            // yazl does not like backslash (\) in the metadata path.
-            const relativePath: string = slash(path.relative(baseDirectoryPath, file));
+            resolve({ isTemporary: true, path: filePath });
+          });
+
+          try {
+            if (compression === 'brotli') {
```

Comment:
```suggestion
            if (compression === 'brotli') {
```

We should create a list of compressions and use from the constant.

Reply:
Changes done in new repo: https://github.com/dream-sports-labs/delivr-cli/commit/2b6ed541be5fe5660506f09c2d1e0f4f26e5c921